packages/sanity/src/core/releases/util/util.ts

```diff
@@ -64,28 +64,12 @@ export function getCreateVersionOrigin(documentId: string): VersionOriginTypes {
 }
 
 /** @internal */
-export function getPublishDateFromRelease(release: ReleaseDocument): Date {
+export function getPublishDateFromRelease(release: ScheduledRelease): Date {
```

Comment:
Updated the type to `ScheduledRelease` to assert that a date will always be returned.